Makefile

```diff
@@ -1,68 +1,161 @@
 help:
 	@echo Deploy rainbow bridge
+	@echo ======================================Local dev=====================================
 	@echo 1 run "make init" first time only and one time.
-	@echo 2 run "make start-bsc" or "make start-ethash"
+	@echo 2 run "make local-start" to start local development tools:ganache, nearup
+	@echo 2 run "make local-start-bsc" to start local development tools:bsc testnet, nearup
 	@echo 3 run "make gen-contarcts"
-	@echo 4 run "make deploy-contarcts"
+	@echo 4 run "make local-full-contracts"
 	@echo 5 run "make start-relayer"
 	@echo 6 run "stop-all"
+	@echo
+	@echo ======================================Testnet======================================
+	@echo 1 run "make init-test-config" copy config testnet file to ${HOME}/.rainbow/config.json
+	@echo 2 run "make testnet-full-contracts" Deploy contracts to BSC and NEAR.
+	@echo 3 run "make start-relayer" Deploy contracts to BSC and NEAR.
+	@echo 4 run "stop-all"
+	@echo
+	@echo ======================================Test the tesnet bridge======================================
+	@echo 1 run "make near-balance" get balance of a near account.
+	@echo 2 run "make transfer-eth-to-near" transfer tokens from eth to near.
+	@echo 3 run "make transfer-near-to-eth" transfer tokens from near to eth.
+	@echo
+	
 
+# ===============================Init==============================
 init: yarn-init gen-contracts
 	
 yarn-init:
 	yarn
 	yarn install
 
+# ===============================Local==============================
+
 # generate ether contracts
 gen-contracts:
 	cd contracts/eth/nearbridge/ && yarn && yarn build
 	cd contracts/eth/nearprover/ && yarn && yarn build
-	
+
 # start near blockchain and connect with ganache.
-start-ethash:
+local-start:
 	cli/index.js clean
 	cli/index.js prepare
-	cli/index.js start near-node
+	cli/index.js start near-node 
 	cli/index.js start ganache
 
 # start near blockchain and connect with binance test net.
-start-bsc:
+local-start-bsc:
 	cli/index.js clean
 	cli/index.js prepare
 	cli/index.js start near-node
 	cli/index.js start binance-smart-chain \
 	--eth-node-url https://data-seed-prebsc-1-s1.binance.org:8545 \
 	--eth-master-sk 0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501200
 
-# deploy contracts
-full-contracts:
-	cli/index.js init-near-contracts
+# deploy full contracts.
+local-full-contracts:
+	cli/index.js init-near-contracts --num-confirmations 3 --near-prover-contract-path ${PWD}/contracts/near/res/bsc_prover.wasm
 	cli/index.js init-eth-ed25519
-	cli/index.js init-eth-client --eth-client-lock-eth-amount 1000 --eth-client-lock-duration 10
+	cli/index.js init-eth-client --eth-client-lock-duration 2 --eth-client-replace-duration 4
 	cli/index.js init-eth-prover
 	cli/index.js init-eth-erc20
 	cli/index.js init-eth-locker
 	cli/index.js init-near-token-factory
```

Comment:
The first word in the command should represent an action. Also, the command itself should be quite self-descriptive without the comment.

And also add another command to clean & prepare like:
```make
setup-clean-and-prepare
    cli/index.js clean
    cli/index.js prepare
```
After this please remove clean&prepare actions from these commands and remove it to:
* start-local-near-and-ganache-nodes
* start-near-and-bsc-nodes (this shouldn't be local as you connect to the external BSC node. Also, those hardcoded values shouldn't be added to the main Makefile)
* deploy-bsc-contracts (this is not local at all as this just using the common config which is not defined in the scope of the command)

Reply:
done